docs/user_guides/advanced/safeguarding-ai-virtual-assistant-blueprint.md

```diff
@@ -0,0 +1,52 @@
+# Safeguarding AI Virtual Assistant NIM Blueprint with NVIDIA NeMo Guardrails and NemoGuard NIMs
```

Comment:
Will this make a very long title in the left navigation bar? If so, alternative title could be "Blueprint with NemoGuard NIMs"